app/assets/javascripts/quby/answers/validation.js

```diff
@@ -221,20 +227,45 @@
     var answered = 0;
     var visible = 0;
     var hidden = 0;
+    var total = 0;
+    var countedKeys = [];
+    var answeredKeys = [];
 
     for(var i = 0; i < groupItems.length; i++){
-      var inputs = $(groupItems[i]).find("input, textarea, select").not(":disabled, :hidden");
+      var item = $(groupItems[i]);
+      var inputs = item.find("input, textarea, select").not(":disabled, :hidden");
+      var countsFor = 1;
+      var answersFor = 1;
+
+      if(item.is('.option')){
```

Comment:
Deze functie wordt zo erg groot en rommelig. 
Wat je ook kunt doen is om te tellen is telkens de keys te pushen naar countedKeys en answeredKeys en daar op het laatst een unique.length op te doen. Ruimt denk ik een stuk op en is duidelijker dan met de countsFor dingen. 

Als ik het zo zie gebruikt de normale scale trouwen niet data-for, maar gewoon for. Dus dat gaat mogelijk fout (zie v_105 Voorbeeldvragenlijst)